mobify/platform-scaffold
# Msg482 switch to different messaging base

Change some Scaffold files so that a non-generator-processed project can use the existing `merlinspotions` Messaging setup.

Most of the changes here are from [#666](https://github.com/mobify/platform-scaffold/pull/666) which I merged to get access to the fixes in it.

 **JIRA**: https://mobify.atlassian.net/browse/MSG-482

## Changes
- Rename the Messaging config to `web/messaging/merlinspotions.yaml`

## How to test-drive this PR
- Clone the repo
- Use Charles to map-local the PWA code from `https://github.com/mobify/duncan/pull/75` to replace the `pwa-messaging-client.js`
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Subscribe by typing `window.Progressive.MessagingClient.subscribe()` in the JS console
- Send a test campaign and verify the message is received
